cli/command/swarm/leave.go

```diff
@@ -19,15 +19,15 @@ func newLeaveCommand(dockerCli *command.DockerCli) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "leave [OPTIONS]",
-		Short: "Leave a swarm",
+		Short: "When run from a worker, leave the swarm.",
```

Comment:
Also, there should be no period at the end for consistency with the other lines.